e2e/sdk.erc721.e2e.test.ts

```diff
@@ -336,5 +336,30 @@ describe('SDK - contract interaction (deploy, load and mint)', () => {
         }),
       ]),
     });
+
+    // test search nfts
+
+    const resultSearch: SearchNftDTO = await sdk.api.searchNfts({
+      query: contractInfo.params.name,
+    });
+
+    expect(resultSearch).toMatchObject({
+      total: expect.any(Number),
+      pageNumber: expect.any(Number),
+      pageSize: expect.any(Number),
+      network: expect.any(String),
+      nfts: expect.arrayContaining([
+        expect.objectContaining({
+          tokenId: expect.any(String),
+          tokenAddress: expect.any(String),
+          metadata: expect.any(String),
+          contractType: expect.any(String),
+          tokenHash: expect.any(String),
+          minterAddress: expect.any(String),
+          blockNumberMinted: expect.any(String),
+          createdAt: expect.any(String),
+        }),
+      ]),
+    });
```

Comment:
I guess the result should contain what was specified in the query (in this case contractInfo.params ), so, can not you make that assertion as well?

Reply:
yeah im already working on that, will push and let u know XD